examples/convention-tag/test-implicit-files

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash
```

Comment:
Not sure I understand what this file is for. Is it part of a larger unit test?

Reply:
Yes. It's called by the `devcheck` script at the top level, which we can use locally and is also invoked by CI.